src/lib/webpack-config.js

```diff
@@ -208,6 +216,37 @@ export default env => {
 			}
 		}),
 
+		// TypeScript
+		customConfig({
+			module: {
+				loaders: [
+					{
+						enforce: 'pre',
+						test: /\.tsx?$/,
+						use: [
+							{
+								loader: resolve(__dirname, './npm-install-loader'),
+								options: {
+									modules: ['typescript', 'awesome-typescript-loader'],
+									save: true
+								}
+							},
+							{
+								loader: 'awesome-typescript-loader',
+								options: {
+									sourceMap: true,
+									useBabel: true,
+									babelOptions: createBabelConfig(env),
```

Comment:
Sorry for not getting back any sooner, I was on vacation the past week. Yep, that's with `useBabel` turned off. I'm not sure why the pipelining fails. My initial assumption was that plugins in webpack enforced via `pre` are executed before everything else and that this would turn `.ts` into `.js` and then the babel loader would match. Somehow this is not the case here. I could try swapping `awesome-typescript-loader` with `ts-loader`. The latter seems to have made huge improvements in the last months.